manual/introduction.md

```diff
@@ -10,14 +10,30 @@ libraryDependencies += "pl.metastack" %%  "metatime" % "%version%"  // Scala
 libraryDependencies += "pl.metastack" %%% "metatime" % "%version%"  // Scala.js
 ```
 
-## Example
-[scala type="section" value="introduction" file="Examples"]
+## Key Features
```

Comment:
I would suggest to keep the features list and create a chapter for each of them. The chapters need to be registered in `Manual` (line 51).